src/modeler/AutoRest.Swagger/Validation/SecurityDefinitionsStructureValidation.cs

```diff
@@ -24,6 +24,11 @@ public class SecurityDefinitionsStructureValidation : TypedRule<Dictionary<strin
         public override ValidationCategory ValidationCategory => ValidationCategory.SDKViolation;
 
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public override ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.ServiceImpactingChanges;
```

Comment:
Is this service impacting? I'm thinking it'll be a swagger update only, do you know how this translated into the SDK generated code?